model/model_training/configs/config.yaml

```diff
@@ -113,41 +113,50 @@ oasst_export_latin_cyrillic:
     - oasst_export:
         lang: "bg,ca,cs,da,de,en,es,fr,hr,hu,it,nl,pl,pt,ro,ru,sl,sr,sv,uk"
         input_file_path: 2023-03-27_oasst_research_ready_synth.jsonl.gz
+    - alpaca
+    - oig_file:
+        source_url: https://huggingface.co/datasets/laion/OIG/resolve/main/unified_chip2.jsonl
+        max_count: 10000
+        min_length: 1000
+        val_split: 0.2
```

Comment:
for 10k samples, val_split=0.1 is probably enough

Reply:
yes definitely, now working on the effective values in other pr.